src/screens/konnectors/LauncherView.js

```diff
@@ -242,6 +242,16 @@ export class LauncherView extends Component {
       ? styles.workerVisible
       : styles.workerHidden
 
+    if (workerVisible) {
+      setFlagshipUI({
+        bottomTheme: 'dark'
+      })
+    } else {
+      setFlagshipUI({
+        bottomTheme: 'light'
+      })
+    }
```

Comment:
Not sure if this is enough, the HomeView can have a white background and so in that cas we should reapply a `dark` bottom theme

@acezard should we use a callerName or call resetUiState here?

Reply:
@acezard ?

Reply:
@Crash-- @Ldoppea in case of doubt, yes `resetUIState` is the best way to handle that since it's linked to the home theme (normal/inverted)